docs/app/pages/components/components-sections/notifications/badge/snippets/app.html

```diff
@@ -0,0 +1,5 @@
+<button class="btn btn-info" [uxBadge]="content" [badgeMaxValue]="maxValue" [badgeOverlap]="overlap"
```

Comment:
I notice a few places are using `btn-info` & `btn-primary` to set the color the buttons. We should use `button-primary` instead. This is a divergence from the bootstrap class name (I don't quite know the reason, this was implement before I start working on the project).

Also I don't believe we actually support `btn-info` any more, I believe this is a legacy style. We should probably stick to one of the button styles here which will help ensure it displays nicely in both documentation sites: https://pages.github.houston.softwaregrp.net/caf/ux-aspects-micro-focus/#/ui-components/buttons